src/tree/networks/NetworkTreeItem.ts

```diff
@@ -4,22 +4,30 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { Network } from "dockerode";
+import { builtInNetworks } from "../../constants";
 import { AzExtParentTreeItem, AzExtTreeItem, IActionContext } from "vscode-azureextensionui";
+import { builtInNetworks } from "../../constants";
 import { ext } from "../../extensionVariables";
 import { callDockerodeWithErrorHandling } from "../../utils/callDockerodeWithErrorHandling";
 import { getThemedIconPath, IconPath } from '../IconPath';
 import { LocalNetworkInfo } from "./LocalNetworkInfo";
 
 export class NetworkTreeItem extends AzExtTreeItem {
-    public static contextValue: string = 'network';
-    public contextValue: string = NetworkTreeItem.contextValue;
+    public static allContextRegExp: RegExp = /Network$/;
+    public static customNetworkRegExp: RegExp = /^customNetwork$/i;
+
     private readonly _item: LocalNetworkInfo;
 
     public constructor(parent: AzExtParentTreeItem, itemInfo: LocalNetworkInfo) {
         super(parent);
         this._item = itemInfo;
     }
 
+    public get contextValue(): string {
+        const prefix = builtInNetworks.includes(this._item.networkName) ? 'default' : 'custom';
```

Comment:
I think it would be faster to do this:

`return builtInNetworks.includes(this._item.networkName) ? 'defaultNetwork' : 'customNetwork';`

This avoids a string concat (and the associated memory alloc, concat, return, etc.). There are a *ton* of things that reference `contextValue` in both the extension code and VS Code itself, so it would be getting called a lot.